packages/amplication-data-service-generator/src/util/get-user-id-type.ts

```diff
@@ -20,6 +20,9 @@ export const getUserIdType = () => {
     throw new Error("User entity must have an id field");
   }
 
-  const { idType } = idField.properties as types.Id;
+  let { idType } = idField.properties as types.Id;
+  if (!idType) {
+    idType = "CUID";
+  }
   return idType;
```

Comment:
Let's try to be more concise by using the nullish coalescing operator

```suggestion
  const { idType } = idField.properties as types.Id;
  return idType ?? "CUID";
```